readme/template-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template
+
+*{Before using this template, read the accompanying README template guide}.*
+
+## {Project} logo and badges
+{This section is optional. Share your logo and add relevant badges to your README.}
+
+## {Project Name} 
+{Optional: Include the project URL and the project owner name}
+
+## Table of contents 
+{This section is optional.}
```

Comment:
As above I would add a short explaining sentence: A table of contents may be useful in the case of lengthy READMEs.